larray/core.py

```diff
@@ -3976,28 +3973,37 @@ def replace_axes(self, axes_to_replace=None, new_axis=None, **kwargs):
          a1 |  3 |  4 |  5
         >>> row = Axis('row', ['r0', 'r1'])
         >>> column = Axis('column', ['c0', 'c1', 'c2'])
-        >>> arr.replace_axes(x.a, row)
+
+        Replace one axis (expect 2 arguments)
```

Comment:
the "expect 2 arguments" part is a bit strange. who should expect? the function or the user? => replace by somethng like:
Replace one axis (the second argument is required when using this syntax)